.github/workflows/build.yml

```diff
@@ -103,8 +105,25 @@ jobs:
         run: |
           docker images discourse/base
 
+      - name: Print compressed summary
+        if: github.event_name == 'pull_request' && matrix.arch == 'amd64'
+        run: |
+          # Push to local repo to compare sizes
+          docker run --quiet -d --rm -p 5002:5000 registry:2
+          sleep 5
```

Comment:
Instead of doing this, I think we can follow https://docs.docker.com/build/ci/github-actions/local-registry/ instead.

Reply:
Oh didn't know services were available to us, very cool!